src/main/resources/application.properties

```diff
@@ -64,4 +64,6 @@ notify.service.host=http://notification-service:9000/
 notify.service.path=v1/notification/send/sync
 
 portal.departmentupdate.path=portal/updateuserroledept
-notification.sender.mail=support@igot-dev.in
\ No newline at end of file
+notification.sender.mail=support@igot-dev.in
+mdo.search.fields=userId,profileDetails.personalDetails.primaryEmail
+mdo.base.url=https://mdo.karmayogi.nic.in/app/approvals/
```

Comment:
provide two variables... with the following values... construct the url in your code.
domain.host=http://localhost
mdo.approval.base.url=/app/approval/{id}/to-approve

domain.host should be provided in deployment script with the proper host address
{id} should be replaced with application id in code.

Reply:
yes implement this